drivers/ethernet/Kconfig.e1000

```diff
@@ -9,7 +9,7 @@
 menuconfig ETH_E1000
 	bool
 	prompt "Intel(R) PRO/1000 Gigabit Ethernet driver"
-	depends on NET_L2_ETHERNET PCI_ENUMERATION
+	depends on NET_L2_ETHERNET && PCI_ENUMERATION
```

Comment:
Please squash this `drivers: eth: e1000: Update to new kconfig syntax` commit to `drivers: eth: e1000: Add driver Intel PRO/1000 Ethernet controller`